src/app_service/service/community/service.nim

```diff
@@ -601,7 +601,7 @@ QtObject:
         self.events.emit(SIGNAL_COMMUNITY_KICKED, CommunityArgs(community: community))
     
     except Exception as e:
-      error "Error handling community updates", msg = e.msg, communities, updatedChats, removedChats
+      error "Error handling community updates", msg = e.msg
```

Comment:
This is indeed the fix for https://github.com/status-im/status-desktop/issues/10628

Reply:
At first I didn't understand you comment, because this change is an obvious fix for #10628.
But then I understood that I made a type in PR description 🙂 I meant that I'm not sure if this PR fixes #10641.

PR description updated.

Reply:
aaaaah, makes more sense :laughing: